### ISSUE TITLE
Exchange Error Notices

### ISSUE THREAD

**Issue by Bitplicity** [swaponline/MultiCurrencyWallet#3051]
Hello,

Please, the exchange page is too wordy and confusing. It should just say something simple like: 

"You do not have enough funds to cover the mining fee (0.1234) and the order amount. Lower the amount to: 0.123 BTC". 

See screenshot:

![Screenshot_20200712-034417_Chrome](https://user-images.githubusercontent.com/66849208/87241667-68be6980-c3f3-11ea-8886-397b1efa9fbe.jpg)

**Comment by Bitplicity** [comment]
Also, I believe the word "maximum" should say "minimum".

**Comment by noxonsu** [comment]
I think this is an orderbook scan issue

**Comment by noxonsu** [comment]
1. If user has 1 eth on his balance and want to exchange the market has only 0.5 eth liquidity, the message should be "please reduce amount to 0.5. This trade amount is too high for present market liquidity.".

**Comment by iower** [comment]
Totally outdated
